docs/architecture/adr-0014-api-documentaton-revisited.md

```diff
@@ -0,0 +1,116 @@
+# API documentation ADR
+## Context
+
+The goal isn't to make this super complicated, but give us a basis for building out more 
+full-fledged documentation as the project goes -- beyond just listing API endpoints 
+(see Auth0's docs section as inspiration https://auth0.com/docs).
+There are javascript libraries that can assist us with parsing the swagger spec,
+so a good amount of the work will be purely presentational.
+
+Of significant import is that the framework that we choose here gives us an opportunity 
+to test out design practices and frameworks before we apply them to our primary applications. 
+Choosing a framework here that we don't plan on using elsewhere would have significant opportunity costs.
+We also need to consider that Raster Foundry is almost certainly going to be transitioned
+over to Angular 2, and this is the only chance we will have to learn about Angular 2 before
+we do so.
+
+### Frameworks to choose from
+#### Spectacle
+##### Pros
+* Very flexible styling
+* Actively maintained, and looks good
+* Delivered assets are a static site
+* Low maintenance
+##### Cons
```

Comment:
This separate header isn't showing up correctly here and in other spots for `Cons` when rendered